### PR TITLE
HARVESTER: upgrade log enhancemnet

### PR DESCRIPTION
**Pull request by WuJun2016**
#### PR Checklist
~- Is this a multi-tenancy feature/bug?
    - [ ] Yes, the relevant RBAC changes are at:~
~- Do we need to backport changes to the [old Rancher UI](https://github.com/rancher/u), such as RKE1?
    - [ ] Yes, the relevant PR is at:~
- Are backend engineers aware of UI changes?
    - [x] Yes, the backend owner is:   @starbops 

Fixes #

- https://github.com/harvester/harvester/issues/3655

### Occurred changes and/or fixed issues

1. When the log download fails, users can see the reason for the failure.
image

### CODE REVIEW

File: pkg/harvester/l10n/en-us.yaml [hunk 1]
```diff
@@ -689,6 +689,7 @@ harvester:
       fail:  Fail
       ongoing: on-going
       downloadLog: Download Log
+      logStatus: Log Donwload Status
```
Comment (futuretea) [inline pkg/harvester/l10n/en-us.yaml:692]: Download

Comment (WuJun2016) [reply pkg/harvester/l10n/en-us.yaml]: Done

**Review by DaiYuzeng — APPROVED** [review]
LGTM!

**Comment by DaiYuzeng** [comment]
@mergify backport release-harvester-v1.2

**Comment by torchiaf** [comment]
@mergify backport release-harvester-v1.1

**Pull request merged**